test/views/StartPageViewSpec.scala

```diff
@@ -110,6 +110,8 @@ class StartPageViewSpec extends AnyWordSpec with Matchers with GuiceOneAppPerSui
       lazy val view: Html = startPageView(isAgent = false, taxYear)(fakeRequest, welshMessages, mockConfig)
       lazy implicit val document: Document = Jsoup.parse(view.body)
 
+      println(document.body().html())
```

Comment:
Just to check should there be a print line here?

Reply:
No there shouldn't be.